xpdtools/tools.py

```diff
@@ -627,41 +630,43 @@ def tallest_peak(g,r):
     return r_val[np.argmax(height)], np.amax(height)
 
 
-def total_counts(g,r):
+def total_counts(g):
     """Returns total number of data points in graph
 
-           Parameters
-           ----------
-           g : ndarray g(r) of the pdf
-           r : ndarray of corresponding r values
-
-           Returns
-           -------
-           float :
-               total number of data points
+       Parameters
+       ----------
+       g : ndarray
+            g(r) of the pdf
 
+       Returns
+       -------
+       int :
+           total number of data points
 
-           """
+    """
     return len(g)
 
 
-def average_pearson(group, g,r):
+def average_pearson(group, g, r):
```

Comment:
Can we also compute the complete pearson correction? (Maybe as a separate function?)

Reply:
I think I may be confused on the stats here, to my knowledge the pearson's coefficients can be used on two different datasets. Is there a separate coefficient to compare a dataset to an entire group of datasets.

Reply:
@Sasaank poke

Reply:
pushed this into the other pr

Reply:
ah ok